viewer/src/utilities/networking/LocalUrlClient.ts

```diff
@@ -6,12 +6,24 @@ import { CadSceneProvider } from '@/dataModels/cad/internal/CadSceneProvider';
 import { CadSectorProvider } from '@/dataModels/cad/internal/sector/CadSectorProvider';
 import { HttpError } from '@cognite/sdk';
 import { ModelUrlProvider } from './types';
+import { EptSceneProvider } from '@/dataModels/pointCloud/internal/EptSceneProvider';
+import { HttpHeadersProvider } from './HttpHeadersProvider';
```

Comment:
Nitpick, but we should try to group imports.

I suggest ordering:
1. External deps, such as ThreeJS and `@cognite/sdk`
2. Other packages, most generic first, i.e. `utilities` before `datamodels`. Group by 'package' and use '@'-imports
3. Imports from the same package, i.e. without '@'-alias